settings.py

```diff
@@ -15,6 +15,8 @@
     }
 }
 
-USE_TZ = False
+DEBUG = True
+
+USE_TZ = "Europe/Kiev"
```

Comment:
The `USE_TZ` setting should be a boolean value (`True` or `False`). If you intend to set the timezone, use the `TIME_ZONE` setting instead. For example, `TIME_ZONE = 'Europe/Kiev'`.